records_mover/db/driver.py

```diff
@@ -79,60 +50,36 @@ def known_supported_records_formats_for_unload(self) -> List[BaseRecordsFormat]:
         with a records target"""
         return []
 
-    def load(self,
-             schema: str,
-             table: str,
-             load_plan: RecordsLoadPlan,
-             directory: RecordsDirectory) -> Optional[int]:
-        """Loads the data from the data specified to the RecordsDirectory
-        instance named 'directory'.  Guarantees a manifest file named
-        'manifest' is written to the target directory pointing to the
-        target records.
-
-        Returns number of rows loaded (if database provides that
-        info).
-        """
-        raise NotImplementedError(f"load not implemented for this database type")
-
-    def can_load_this_format(self, source_records_format: BaseRecordsFormat) -> bool:
-        """Return true if the specified format is compatible with the load()
-        method"""
-        return False
 
-    def known_supported_records_formats_for_load(self) -> List[BaseRecordsFormat]:
-        """Candidates to look through when negotiating a common records format
-        with a records source.  Will be looked through in order, so
-        the better formats (higher fidelity and/or more efficient to
-        process) should appear first in the list.
-        """
-        return []
+# TODO: remove UnloadsToRecordsDirectory
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker